lib/logger/lib/logger.ex

```diff
@@ -19,6 +19,27 @@ defmodule Logger do
     * Wraps OTP's `error_logger` to prevent it from
       overflowing.
 
+  A common use case is tracking when an important function is called.
```

Comment:
Good point. How's this?

>   Logging is useful for tracking when an event of interest happens in
  your system. For example, logging incoming synchronous calls in a
  `GenServer` behaviour can provide valuable insight.

      def handle_call(request, from, state) do
        Logger.info fn ->
          "handle_call/3: #{inspect request} request received " <>
          "from #{inspect from} with state #{inspect state}."
        end
        # ...
      end